src/dmd/mtype.h

```diff
@@ -180,6 +182,7 @@ class Type : public ASTNode
     static Type *tdstring;              // immutable(dchar)[]
     static Type *terror;                // for error recovery
     static Type *tnull;                 // for null type
+    static Type *tnoreturn;             // for bottom type typeof(*null)
```

Comment:
Why is this added to `stable`. The implementation of bottom types is currently incomplete in `master` and non-existent in `stable`. Shouldn't we wait until bottom types are fully implemented in master?

Reply:
I've unexpectedly encountered it in the codegen AST, so it's definitely in stable. IIRC, when compiling `core.internal.abort`.

Reply:
Because we keep on getting piece-meal PRs such as https://github.com/dlang/druntime/pull/3381

Reply:
But all those PRs were targeting master, how did bits end up in stable?

Reply:
Because it's 2.096.0beta.

Reply:
Got it.